### DISCUSSION TITLE
Is there an easy way to maintain changelog/history on models on every create/update/delete?

### DISCUSSION THREAD

**Discussion by kuldeepk** [prisma/prisma#12205] [Q&A] [answered]
In Django there is a [library](https://django-simple-history.readthedocs.io/en/latest/) which allows one to easily maintain history on models for every create/update/delete? Is there a similar library or a way to support that in Prisma?

**Comment by austincrim** [comment, marked as answer]
Hey @kuldeepk 👋 ,

We do not have an official solution for this. You could most likely build what you are looking for using [Prisma middleware](https://www.prisma.io/docs/concepts/components/prisma-client/middleware), but if you have an idea of how we could solve this, please take a look at [this existing issue](https://github.com/prisma/prisma/issues/1902), or if you feel your requirements are unique, outline a new [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=).

Thanks!

**Reply by kuldeepk** [reply]
Thanks @austincrim! [This](https://github.com/prisma/prisma/issues/1902) issue is exactly what I was getting at. Is there any timeline for that?

**Reply by austincrim** [reply]
I don't think we have any official timeline, but any updates will happen in that issue!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
